SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -44,11 +44,14 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKSchemaGenerator);
 TArray<UClass*> SchemaGeneratedClasses;
 TMap<FString, FActorSchemaData> ActorClassPathToSchema;
 TMap<FString, FSubobjectSchemaData> SubobjectClassPathToSchema;
-uint32 NextAvailableComponentId = SpatialConstants::STARTING_GENERATED_COMPONENT_ID;
+Worker_ComponentId NextAvailableComponentId = SpatialConstants::STARTING_GENERATED_COMPONENT_ID;
+
+// Sets of data/owner only/handover components
+TMap<ESchemaComponentType, TArray<Worker_ComponentId>> SchemaComponentTypeToComponents;
 
 // LevelStreaming
-TMap<FString, uint32> LevelPathToComponentId;
-TSet<uint32> LevelComponentIds;
+TMap<FString, Worker_ComponentId> LevelPathToComponentId;
+TArray<uint32> LevelComponentIds;
```

Comment:
Use `Worker_ComponentId` instead?

Reply:
Wait - why was `Worker_ComponentId ` added in the first place?

Reply:
This one I tried it as worker component ID, but assigning the array to the array of uints in the schema database would not compile- this could well be something I've messed up though, so let me know if this should be possible
if you mean why did I add worker_componentID above, it's because it is a component ID and I was changing uint32's to component IDs based on another comment above, happy to revert that if they shouldn't be here

Reply:
Seems legit to me @m-samiec?

Reply:
Should be fine to assign it within the schemadatabase. Why was it changes from a set to an array?

Reply:
It should be a set, that's my bad- reverting this

Reply:
reverted